.travis.yml

```diff
@@ -107,6 +107,8 @@ install:
   - conda create --yes --quiet -n statsmodels-test ${PKGS} ${OPTIONAL} nose pip pyyaml setuptools
   - source activate statsmodels-test
   - if [ ${COVERAGE} = true ]; then pip install coverage coveralls; fi
+  - pip install pip --upgrade
+  - pip install patsy --upgrade --upgrade-strategy only-if-needed
```

Comment:
but this will upgrade patsy from the pinned version, does it.

I was just preparing a reduced version requirement for patsy, until I got puzzled why pandas didn't fail